README.md

```diff
@@ -12,6 +12,11 @@ To install the requirements first run the command
 ```
 sudo pip install -r requirements.txt
 ```
+For Windows users, the word "sudo" should not be used. Hence, you should run the command:
```

Comment:
@abhsag24 Ok, so should I change it and send a PR again, or have you already corrected it and closed the issue?